### PR TITLE
added stickermule sponsor

### LINKED ISSUE #20: [Home page] Add Sponsors section
All of our hackathon sponsors should be displayed on our Home page
- Include logos and links in sponsors section
- Provide sponsor information through reusable format (e.g. `src/assets/data/sponsors.json`)
- Include effect to slightly enlarge logo on hover

### PR DESCRIPTION
**Pull request by Pegysus**
- Added Stickermule logo and link to their website in a Sponsor section
- Added information for other sponsors in sponsors.json but still have yet to implement their links on the section

### CODE REVIEW

**Comment by rileysw** [comment]
This is a temporary solution since the deadline to publish the Sticker Mule logo is by Wednesday morning. The temporary design implementation (as well as the data in sponsors.json) was approved by Parsa.

I think both @Pegysus and me were confused about using the image relative paths from sponsors.json since it wouldn't load correctly into the image component.

File: src/views/home/sections/Sponsors/Sponsors.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import sponsors from "assets/data/sponsors.json";
+
+import styles from "./Sponsors.module.scss";
+
+import stickerMule from "assets/sponsors/sticker_mule.svg";
+
+function SponsorItem(props: any) {
+	return (
+		<Link href={props.data.url}>
```
Comment (taesungh) [inline src/views/home/sections/Sponsors/Sponsors.tsx:14]: Suggestion: `next/link` should be used only for internal routing, not to link to external sites.

File: src/views/home/sections/Sponsors/Sponsors.module.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+.smallTierImage {
+	width: 200px;
+	height: 200px;
+}
+
+.smallTierImage:hover {
+	transform: scale(1.1);
+	transition: all 0.3s ease-in-out;
+}
```
Comment (taesungh) [inline src/views/home/sections/Sponsors/Sponsors.module.scss:14]: Suggestion: specifiers can be nested with SASS.

**Review by taesungh — COMMENTED** [review]
Few more things to address, I'm working on most of them.

File: src/views/home/sections/Sponsors/Sponsors.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+import { Col, Row } from "react-bootstrap";
+
+import sponsors from "assets/data/sponsors.json";
+
+import styles from "./Sponsors.module.scss";
+
+import stickerMule from "assets/sponsors/sticker_mule.svg";
+
+function SponsorItem(props: any) {
```
[the remark this answers is not in the document]
Comment (taesungh) [reply src/views/home/sections/Sponsors/Sponsors.tsx]: `SponsorItemProps` added in 0511137.

File: src/views/home/sections/Sponsors/Sponsors.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+function Sponsors() {
+	// Temporary workaround
+	const stickerMule = sponsors.smallTier[4];
+
+	return (
+		<section id="sponsors" className="container">
+			<h2>Sponsors</h2>
+			<p className={styles.sponsorInfo}>
+				Interested in sponsoring HackUCI 2023? Check out our information above
+				to learn more about our event! For more information, please email us
```
Comment (taesungh) [inline src/views/home/sections/Sponsors/Sponsors.tsx:40]: Suggestion: use of word "information" seems a little redundant/conflicting. Perhaps say "For additional inquiries" in second sentence.

File: src/views/home/sections/Sponsors/Sponsors.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+	return (
+		<section id="sponsors" className="container">
+			<h2>Sponsors</h2>
+			<p className={styles.sponsorInfo}>
+				Interested in sponsoring HackUCI 2023? Check out our information above
+				to learn more about our event! For additional inquiries, please email us
+				at&nbsp;
+				<a href="mailto: hack@uci.edu">hack@uci.edu</a>.
+			</p>
```
Comment (samderanova) [inline src/views/home/sections/Sponsors/Sponsors.tsx:43]: Nitpick: We should center the paragraph element and change the color of the link to something other than blue (perhaps same orange as the link in the footer) for consistency.

File: src/views/home/sections/Sponsors/Sponsors.tsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+			<Row className="my-4">
+				<Col className={styles.column}>
+					<SponsorItem {...stickerMule} src={logo_stickerMule} />
+				</Col>
```
Comment (samderanova) [inline src/views/home/sections/Sponsors/Sponsors.tsx:47]: Awesome animation!

**Review by samderanova — CHANGES_REQUESTED** [review]
Thanks for working on this! Just have one small suggestion.

**Comment by taesungh** [comment]
Please fix the formatting/linting errors rather than dismissing them.

**Review by samderanova — APPROVED** [review]

**Review by taesungh — APPROVED** [review]
Thanks for starting this. Temporary solution works for now; we'll clean up more later.

**Pull request merged**